Dockerfile

```diff
@@ -324,4 +345,4 @@ COPY rootfs/make_iso.sh /
 
 RUN /make_iso.sh
 
-CMD ["cat", "boot2docker.iso"]
+CMD ["sh", "-c", "[ -t 1 ] && exec bash || exec cat boot2docker.iso"]
```

Comment:
This one is so that our existing flow of `docker run --rm boot2docker/boot2docker > boot2docker.iso` is still unchanged, but doing `docker run -it --rm boot2docker/boot2docker` doesn't start outputting an ISO...  :smile:  (NOT AGAIN :sob:  *slams the mute keystroke*)